plugins/copy-to-clipboard/prism-copy-to-clipboard.js

```diff
@@ -10,9 +10,23 @@
 	}
 
 	var Clipboard = window.Clipboard || undefined;
+	var _nativeClipboard;
+
+	try {
+		// Try using something ClipboardJS-specific
+		new Clipboard('').destroy();
```

Comment:
You probably don't need the `.destroy()` bit cuz `new Clipboard` will throw on its own, so if we `new` and it doesn't throw, we _should_ be able to assume it's not the global version? Unless `window.Clipboard` becomes instantiable.

Although we're kind of fighting against the future, trying to guess what the global Clipboard will be capable of...

Reply:
Well, yeah I know... It's supposed to be a temporary fix, anyway. I hope ClipboardJS will release a non-conflicting version soon, so we can properly rollback this.